src/features/swap/index.tsx

```diff
@@ -124,7 +126,7 @@ const SwapWidget = ({ sell }: Params) => {
       height: '860px',
       // provider: safeAppWeb3Provider, // Ethereum EIP-1193 provider. For a quick test, you can pass `window.ethereum`, but consider using something like https://web3modal.com
       chainId, // 1 (Mainnet), 5 (Goerli), 100 (Gnosis)
-      // standaloneMode: false,
+      standaloneMode: true,
```

Comment:
This should be false. If it is true the app displays a button to connect to the widget, we don't want this since we want the users to connect through our wallet connectors

Reply:
The default when not set is "false", but Anxo was suggesting to be explicit and have it set to "false"

Reply:
Ah true it was the opposite! Thanks, I just changed it. Now its also not possible anymore to "Disconnect" from the widget which is good.